src/Symfony/Component/HttpKernel/Attribute/HttpException/AccessDenied.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Attribute\HttpException;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Attribute\HttpException;
+
+/**
+ * @author Dejan Angelov <angelovdejan@protonmail.com>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS)]
+class AccessDenied extends HttpException
+{
+    public function __construct(array $headers = [])
+    {
+        parent::__construct(
+            Response::HTTP_FORBIDDEN,
```

Comment:
I prefer using integers here. HTTP is a standard, the consts add an extra unneeded indirection.
Shouldn't the name of the class match the name of the corresponding status code BTW?

Reply:
I used `AccessDenied` as it's used for the `\Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException` exception as well